packages/app/src/app/pages/Dashboard/Components/Repository/RepositoryListItem.tsx

```diff
@@ -65,40 +65,43 @@ export const RepositoryListItem: React.FC<RepositoryProps> = ({
         {...props}
       >
         <Grid css={{ width: 'calc(100% - 26px - 8px)' }} columnGap={4}>
-          <Column
-            span={[10, 5, 4]}
-            css={{
-              display: 'block',
-              overflow: 'hidden',
-              paddingBottom: 4,
-              paddingTop: 4,
-            }}
-          >
-            <Stack gap={4} align="center" marginLeft={2}>
-              <Icon color="#999" name="repository" size={16} width="32px" />
-              <Element css={{ overflow: 'hidden' }}>
-                <Text
-                  size={3}
-                  weight="medium"
-                  maxWidth="100%"
-                  css={{ color: isViewOnly ? '#999999' : '#E5E5E5' }}
-                >
-                  {repository.owner}/{repository.name}
-                </Text>
-              </Element>
+          <Column span={[12, 8, 8]}>
+            <Stack gap={3} align="center" marginLeft={2}>
+              <Text
+                css={{
+                  fontSize: '14px',
+                  color: isViewOnly ? '#999999' : '#E5E5E5',
+                }}
+              >
+                {repository.name}
+              </Text>
+              <Text size={2} color="#999">
+                {repository.owner}
+              </Text>
             </Stack>
           </Column>
-          <Column span={[0, 2, 2]}>
-            {isViewOnly ? (
-              <Stack align="center">
-                <Badge variant="trial">View only</Badge>
+          <Column span={[0, 4, 4]} as={Stack} align="center">
+            <Stack
+              justify="flex-end"
+              align="center"
+              gap={2}
+              css={{ paddingRight: '16px' }}
+            >
+              {isViewOnly ? (
+                <Stack align="center">
+                  <Badge variant="trial">Restricted</Badge>
+                </Stack>
+              ) : null}
+              {repository.private ? (
+                <Icon color="#808080" name="lock" size={12} />
+              ) : null}
+              <Stack gap={1}>
+                <Icon color="#999" name="branch" size={12} />
+                <Text size={3} variant="muted" maxWidth="100%">
+                  {labels.branches}
```

Comment:
When the amount of `branches` is three digits and the amount of `branches` in the repo above or below are less digits, the lock icons are no longer aligned with each other. Not sure if we want those to be columns too, or if it's ok that they're not aligned?

cc @lenasmayor 

<img width="888" alt="image" src="https://user-images.githubusercontent.com/7533849/204745236-f6f47a39-9cc7-48a7-9a4b-da0388948a5e.png">

Reply:
Hmm.. that's a tricky one, I'd say it's ok to keep it this way otherwise we will have to create a new logic just for the list page which I don't think justifies the effort.